packages/styles/scss/patterns/sub-patterns/link-list/_link-list.scss

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@import '../../../globals/utils/hang';
```

Comment:
One thing, if a user were to pull just the `LinkList` in isolation, this should also pull in any styles necessary (e.g. CTA styles)

Reply:
I believe they need Card pattern scss as well.

Reply:
Right, so those would need to be added here, or if we want to follow other patterns, we create an `index.scss` file under `/scss/patterns/sub-patterns/link-list/` and include all the necessary style includes including `link-list.scss`. The READMEs will need to be updated too to point to the `index.scss` file instead of `list-list.scss`.

Reply:
Okay, will update.

Reply:
@jeffchew updated the PR with above changes.